src/docfx.website.themes/common/RestApi.common.js

```diff
@@ -246,3 +250,23 @@ exports.transform = function (model) {
         return path;
     }
 }
+
+exports.getBookmarks = function (model) {
+    if (!model) return null;
+
+    var bookmarks = {};
+
+    bookmarks[model.uid] = "";
+    if (model.tags) {
+        model.tags.forEach(function (tag) {
+            bookmarks[tag.uid] = tag["x-bookmark-id"] ? tag["x-bookmark-id"] : common.getHtmlId(tag.uid);
+        });
+    }
+    if (model.children) {
+        model.children.forEach(function (child) {
+            bookmarks[child.uid] = common.getHtmlId(child.uid);
```

Comment:
>uid [](start = 58, length = 3)

for docs template, we can use common.getHtmlId(operationId) as bookmark, and not enable tag bookmark since they use Noun_Verb, thanks!